components/pedigree/resources/src/main/resources/pedigree/view/person.js

```diff
@@ -1505,6 +1544,13 @@ define([
                 } else {
                     this.setLifeStatus('alive');
                 }
+                if(info.hasOwnProperty('aliveandwell')) {
+                    if (!this.getAliveAndWell()) {
```

Comment:
this should be something ismilar to how"evaluated status" is set below:

```
if (this.getAliveAndWell() != info.aliveandwell) {
   this.setAliveAndWell(info.aliveandwell);
}
```

...the point is to set only if it is different from what it is set to